### ISSUE TITLE
Version mismatch. Client at 208 while server at 207.

### ISSUE THREAD

**Issue by zakiaziz** [facebook/nuclide#1035]
### Issue and Steps to Reproduce
I've installed the apm packing locally on my machine, and the npm on the remote server. I also installed watchman and I put in the relevent connection info in the remote connection window and hit connect. This error popped up:
![screen shot 2017-02-27 at 10 36 23 pm](https://cloud.githubusercontent.com/assets/940281/23390644/6db082fa-fd3d-11e6-8222-4a3db182417a.png)

### Expected Behavior
Should have connected to the remote server.

### Actual Behavior
Did not connect to remote server. Error message displayed.

### Versions
- Atom: v1.14.3
- Nuclide: v0.207.0
- Client OS: OSX 10.12.2
- Server OS (optional): Ubuntu 14.04

### Additional Details
* Installed packages (`apm ls --installed`):
```
Community Packages (1) /Users/zakiaziz/.atom/packages
└── nuclide@0.207.0
```

**Comment by johnislarry** [comment]
Hey @zakiaziz I wonder if it's possible that atom auto-upgraded you to Nuclide's v0.208.0 (which was just published a few hours ago) on the client, and this mismatched your server version.  It seems like upgrading the `nuclide` npm package on your server to v0.208.0 will fix the issue.

**Comment by zakiaziz** [comment]
Just updated the package, still getting the same error
![screen shot 2017-02-28 at 12 58 24 am](https://cloud.githubusercontent.com/assets/940281/23393474/15fc160a-fd51-11e6-983b-3a079d366204.png)

**Comment by johnislarry** [comment]
This looks like `nuclide` is a dependency of a different package.  Were these the steps you followed?

https://nuclide.io/docs/features/remote/#nuclide-server__setup

**Comment by zakiaziz** [comment]
Yes I followed those steps. I don't think Nuclide is a dependency of a different package, here is the full out put of `npm list -g`: https://gist.github.com/zakiaziz/27d69cba93c09adb917c5057f822c43e

**Comment by johnislarry** [comment]
I wonder if the old Nuclide server is running on your server machine?  Can you check for rogue node or nuclide processes and kill them?

**Comment by zakiaziz** [comment]
I've tried
- `killall nuclide`
- `killall node`
- `ps aux | grep node` -> `kill -9 PID` whatever PID had nuclide in it
- uninstall and reinstall `npm uninstall -g nuclide`, `rm -rf /usr/lib/node_modules/nuclide/` -> `npm install -g nuclide`

At this point I'm able to `nuclide-start-server` on my server and get a json response. But when I go back to atom and try I see this:
![screen shot 2017-03-03 at 4 34 36 pm](https://cloud.githubusercontent.com/assets/940281/23570757/c86c9244-0033-11e7-8b3d-19542ed770af.png)

**Comment by johnislarry** [comment]
I think this is because of how you're starting the server manually.  Can you try killing the nuclide server on your server machine, and then add a remote project from the Nuclide client?  That should cause Nuclide to spin the server up by itself automatically.

**Comment by zakiaziz** [comment]
Same issue
```
zaki@server ~ $ ps aux | grep nuclide
zaki     10523  0.0  0.2  11748  2140 pts/1    S+   00:37   0:00 grep nuclide
```

I've also tried installing nuclide (and watchman) on a different server and i get the exact same error

**Comment by iNCurrO** [comment]
although my msg is "Version mismatch. Client at 213 while server at 210." (different version),
I have same err. 
I also tried all of methods which are used by @zakiaziz

**Comment by vijayg78** [comment]
me too have the same error. 
The server successfully started, but we were unable to connect.
Original error message:
 Version mismatch. Client at 242 while server at 240.

I have reinstalled nuclide on server which is 242, but still when i start the server (stopall and start) it is starting with version 240. What might be the issue?

**Comment by shengxuanye** [comment]
@vijayg78 
`sudo npm remove nuclide -g` and `sudo npm install nuclide -g` works for me.

**Comment by vijayg78** [comment]
for me still it does not work. Client atom is at 242. At the server (ubuntu, AWS) i stopped the service, removed it and installed it. while installation it said version 242. But somehow the service is still returning 240 (nuclide-start-server).
